src/App.js

```diff
@@ -1,5 +1,70 @@
+import { Console, MissionUtils } from "@woowacourse/mission-utils";
+import {
+  inputCarsException,
+  inputCountException,
+} from "./exceptionHandling.js";
+
 class App {
-  async run() {}
+  async run() {
+    try {
+      const cars = await inputCars();
+      const count = await inputCount();
+      const result = await racingCount(count, cars);
+
+      Console.print("최종 우승자 : " + getWinners(cars, result));
```

Comment:
```suggestion
      Console.print(`최종 우승자 : ${getWinners(cars, result)}`);
```
Airbnb JS 스타일 가이드에서는 문자열을 연결할 때 template string 사용을 권장하는 것 같습니다!

https://github.com/airbnb/javascript?tab=readme-ov-file#es6-template-literals

Reply:
Airbmb 스타일 가이드 숙지를 많이 해봐야겠네요
위에 중곤씨의 멋진 블로그를 보며.. airbnb와 prettier 적용을 바로 해보도록하겠습니다😁